### DISCUSSION TITLE
Suggestion: Lasso tool for more precise selections

### DISCUSSION THREAD

**Discussion by carolinaqzx** [excalidraw/excalidraw#5263] [General]
I am starting to use Excalidraw as a bullet journal on my iPad, so I usually hand write my notes, rather than type them. I often have to move around text or images that I draw, but I have a difficult time selecting them properly, especially if the strokes of one word are too close to another word. I would really appreciate the incorporation of a lasso tool for more precise selections.

**Comment by aidan-gibson** [comment]
This feature is absolutely essential for those of us using tablets with a stylus! GoodNotes, Notability, and Apple Notes all have some version of a "lasso" feature. This is the one thing preventing me from using excalidraw, and I hate using closed-source apps ):
